new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/InstallParams.java

```diff
@@ -53,7 +51,7 @@ public InstallParams version(String version) {
 
   public String version() {
     if (version == null) {
-      return WIT_TYPE.equals(type) ? DEFAULT_WIT_VERSION : DEFAULT_WDT_VERSION;
+      return WIT.equals(type) ? WIT_VERSION : WDT_VERSION;
```

Comment:
this is ok, but i had to stop and think to work out what you meant here.
i know this is uglier:
``` 
if (type == WIT) {
  return WIT_VERSION;
} else if (type == WDT) {
  return WDT_VERSION;
} else {
 // do something!!!
}
```

but it is much easier to read and immediately understand what you meant, plus it is more explicit about what do to if type is not WIT or WDT ..

Reply:
will change to 
return WIT.equals(type) ? WIT_FILE_NAME : WDT_FILE_NAME;

Reply:
Where there are this many "if" checks on type, it makes me think this should be two separate classes (that could extend the same base/interface).  It would also make it easier to read :)